src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -232,10 +232,13 @@ def geoprocess(endpoint, data, retry=None):
     try:
         response = requests.post(geop_url,
                                  data=json.dumps(data),
-                                 headers={'Content-Type': 'application/json'})
+                                 headers={'Content-Type': 'application/json'},
+                                 timeout=200)
     except ConnectionError as exc:
         if retry is not None:
             retry(exc=exc)
+    except Timeout as exc:
+        raise Exception('Geoprocessing service timed out: {}'.format(exc))
```

Comment:
Is there a way to report the actual exception contents in the job table but restrict what gets sent to the client?  The timeout exception reports internal hosts and ports which while not crucial to protect does feel like an unnecessary leak.  It's likely that the qualitative description you provide is more meaningful than the `Timeout` exception contents in the case where we have to choose between leaking and not capturing that exception detail.

Reply:
I'll look into splitting our job error and what gets sent back to the client, though I'm not sure we're set up for that. We could also write details we don't want to send to the client to the celery log instead. I agree they're not even that helpful, so I may just remove the printing of the exception entirely.